packages/node/src/utils/normalize.spec.ts

```diff
@@ -52,6 +52,28 @@ describe('normalizeBuildOptions', () => {
     expect(result.main).toEqual('/root/apps/nodeapp/src/main.ts');
   });
 
+  it('should resolve additional entries from root', () => {
+    const result = normalizeBuildOptions(
+      {
+        ...testOptions,
+        additionalEntryPoints: [
```

Comment:
I'm not sure on the opinion of the rest of the devs here, but would it be better to add the `addtionalEntryPoints` to the top level `beforeEach` `testOptions` instead?

Reply:
I think it's fine here. The `additionalEntryPoints` is an _optional_ property anyway, so having it here makes it so that the other tests behave as before.